modules/odc-using-web-terminal.adoc

```diff
@@ -18,4 +18,17 @@ After the Web Terminal Operator is installed, you can use the web terminal as fo
 +
 . Click *Start* to initialize the web terminal using the selected project.
 
++
 After the web terminal is initialized, you can use the preinstalled CLI tools like `oc`, `kubectl`, `odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx` in the web terminal.
++
+
+. Click *+* to open multiple tabs within web terminal in the console.
+
++
+[NOTE]
+====
+* You can re-run commands by selecting them from the list of commands you have run in the terminal. These commands persist across multiple terminal sessions.
+
+* The web terminal remains open until you close it or you close the browser window/tab.
```

Comment:
```suggestion
* The web terminal remains open until you close it or you close the browser window or tab.
```

Reply:
Overall, very crisp and clear. Great work.
Red Hat prefers that we use "or" instead of "/".